packages/input-select/src/Select.js

```diff
@@ -68,6 +68,7 @@ class Select extends React.Component {
       this.input.select();
     }
     this.setState({ isMenuOpen });
+    this.props.onMenuVisibilityChange?.(isMenuOpen);
```

Comment:
LGTM. Just add an action to the stories and we're good to go 😃

Reply:
Done! I ammended the change on the previous commit